test/server/graphql/loaders/expense.test.js

```diff
@@ -30,7 +30,7 @@ describe('server/graphql/loaders/expense', () => {
     otherPayoutMethod = await fakePayoutMethod({ type: 'OTHER' });
   });
 
-  describe('userTaxFormRequiredBeforePayment', () => {
```

Comment:
These two tests were pointing to the two loaders that we've merged above. They're testing different scenarios that would be difficult to merge, so I instead named them `userTaxFormRequiredBeforePayment (1)` and `userTaxFormRequiredBeforePayment (2)`. Keeping the same `describe` structure for a cleaner diff.